beacon-chain/rpc/prysm/v1alpha1/beacon/validators.go

```diff
@@ -537,6 +537,15 @@ func (bs *Server) GetValidatorParticipation(
 		if err != nil {
 			return nil, status.Errorf(codes.Internal, "Could not pre compute attestations: %v", err)
 		}
+	case version.Bellatrix:
+		v, b, err = altair.InitializePrecomputeValidators(ctx, beaconState)
```

Comment:
I suggest to add a comment that Bellatrix precompute is equal to Altair precompute for future readers